src/lib/Lib/MrDocsCompilationDatabase.hpp

```diff
@@ -19,6 +19,12 @@
 namespace clang {
 namespace mrdocs {
 
+/**
+ * Execute cmake to export compile_commands.json.
+*/
+std::optional<std::string> 
+executeCmakeExportCompileCommands(llvm::StringRef cmakePath, llvm::StringRef cmakeListsPath);
```

Comment:
This functionality seems unrelated. `MrDocsCompilationDatabase` requires an existing database it's going to convert. There can be no existing database if there's no `compile_commands.json` yet. So `MrDocsCompilationDatabase` can't be using this function.